app/renderer/js/pages/preference/general-section.ts

```diff
@@ -389,11 +390,78 @@ export default class GeneralSection extends BaseSection {
 		}
 	}
 
-	updateResetDataOption(): void {
-		const resetDataButton = document.querySelector('#resetdata-option .reset-data-button');
+	/* Reset App Data */
+	resetAppData(): void {
+		const resetAppButton = document.querySelector('#resetdata-option .reset-data-button');
+		resetAppButton.addEventListener('click', () => {
+			this.resetAppSettings();
+		});
+	}
+
+	async resetAppSettings(): Promise<void> {
+		const resetMessage = 'Selecting YES will reset all the settings to default without removing any connected organizations and preferences from Zulip.';
```

Comment:
We are removing preferences here. So should no write that preferences will not be removed.

Reply:
My bad. That should be organisations and accounts